masterlist.yaml

```diff
@@ -121,6 +121,14 @@ common:
   - &dirtyPlugin
     util: '[SSEEdit](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
     info: 'A cleaning guide is available [here](https://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit).'
+  - &quickClean
+    util: '[SSEEdit Experimental](https://github.com/TES5Edit/TES5Edit/releases)'
+    info: |+
+
+          > **Quick Clean** is a xEdit startup option which will automatically clean a selected plugin.  
```

Comment:
Not exactly sure about using a double space to force a newline on these block qoutes.

```yaml
    info: |

      >**Quick Clean** is a xEdit startup option which will automatically clean a selected plugin.
      >
      >A guide for running **Quick Clean** can be found [here](https://tes5edit.github.io/docs/5-mod-cleaning-and-error-checking.html#Apendix-C). 
      >
      >The latest version of [SSEEdit Experimental](https://github.com/TES5Edit/TES5Edit/releases) is required to use **Quick Clean**.
```
I could use that instead.

Reply:
It was mainly to give it some space and make it clearer to the reader.
![image](https://user-images.githubusercontent.com/1944639/47252991-44637200-d445-11e8-904b-07f461dd3af8.png)